src/components/ChatForm/useCheckBoxes.ts

```diff
@@ -54,26 +56,17 @@ const useAttachActiveFile = (
   });
 
   useEffect(() => {
-    if (!attachFileCheckboxData.checked || !interacted) {
-      setAttachFile((prev) => {
-        return {
-          ...prev,
-          hide: !shouldShow,
-          value: filePathWithLines,
-          disabled: !activeFile.name,
-          fileName: activeFile.name,
-          checked: !!activeFile.name && shouldShow && hasSnippet && !interacted,
-        };
-      });
-    }
-  }, [
-    activeFile.name,
-    attachFileCheckboxData.checked,
-    filePathWithLines,
-    hasSnippet,
-    interacted,
-    shouldShow,
-  ]);
+    setAttachFile((prev) => {
```

Comment:
Will this update after the user has checked the box?

Reply:
Yes it will, I think that might be an issue, since when the user clicks attach they won't want the file to change (this also lets them add a different snippet)

Reply:
Oh, okay, I was thinking that it's an issue that file in not reattaching

But than we have another issue: line selection is not changing relatively to cursor position.